lib/activerecord-import/adapters/postgresql_adapter.rb

```diff
@@ -21,7 +21,7 @@ def insert_many( sql, values, *args ) # :nodoc:
       insert( sql2insert, *args )
     end
 
-    ActiveRecord::Base.connection.query_cache.clear
+    query_cache.clear
```

Comment:
@KentoMoriwaki could you explain why this line of code is needed? I'm trying to make sure there is test coverage. I found this test, but still don't really understand the purpose:

https://github.com/zdennis/activerecord-import/blob/master/test/support/postgresql/import_examples.rb#L23

Thanks!